lib/src/components/NxThreatIndicator/__tests__/NxThreatIndicator.test.tsx

```diff
@@ -4,116 +4,115 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import { faCircle } from '@fortawesome/free-solid-svg-icons';
-
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
-import 'jest-enzyme';
-
-import NxThreatIndicator from '../NxThreatIndicator';
-import NxFontAwesomeIcon from '../../NxFontAwesomeIcon/NxFontAwesomeIcon';
-import NxToolip from '../../NxTooltip/NxTooltip';
+import NxThreatIndicator, { Props } from '../NxThreatIndicator';
+import { rtlRender, userEvent } from '../../../__testutils__/rtlUtils';
+import { RenderResult, screen } from '@testing-library/react';
+import {
+  allThreatLevelCategories,
+  allThreatLevelNumbers,
+  categoryByPolicyThreatLevel
+} from '../../../util/threatLevels';
 
 describe('NxThreatIndicator', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent(NxThreatIndicator, {});
-
-  it('renders an faCircle NxFontAwesomeIcon with the `nx-threat-indicator` class', function() {
-    const icon = getShallowComponent().find(NxFontAwesomeIcon);
-
-    expect(icon).toHaveClassName('nx-threat-indicator');
-    expect(icon).toExist();
-    expect(icon).toHaveProp('icon', faCircle);
-  });
-
-  it('sets the nx-threat-indicator--unspecified class if no props are passed', function() {
-    const icon = getShallowComponent().find(NxFontAwesomeIcon);
+  const quickRender = rtlRender<Props>(NxThreatIndicator, {});
 
-    expect(icon).toMatchSelector('.nx-threat-indicator--unspecified');
+  it('renders a threat indicator', function() {
+    const el = quickRender(),
+        threatIndicator = el.getByRole('img', { hidden: true, queryFallbacks: true });
```

Comment:
Is there a reason you're not testing the `graphics-symbol` role?

Reply:
> Is there a reason you're not testing the `graphics-symbol` role?

Since we're using `queryFallbacks` and since `graphics-symbol` is a subclass of `img`, I thought it'd be safer to use the `img` role.

Reply:
> Since we're using queryFallbacks and since graphics-symbol is a subclass of img, I thought it'd be safer to use the img role.

It's safe in that it certainly enables the tests to pass but it means that they aren't as complete; they aren't testing that the `graphics-symbol` role is present

Reply:
> > Since we're using queryFallbacks and since graphics-symbol is a subclass of img, I thought it'd be safer to use the img role.
> 
> It's safe in that it certainly enables the tests to pass but it means that they aren't as complete; they aren't testing that the `graphics-symbol` role is present

Changed to use `graphics-symbol`. With this though, I don't think we need the `{ queryFallbacks: true }` anymore either.

Reply:
> With this though, I don't think we need the { queryFallbacks: true } anymore either.

Right I was expecting that to be removed.  I also think we should have one test that is about the fallback to the `img` role

Reply:
> > With this though, I don't think we need the { queryFallbacks: true } anymore either.
> 
> Right I was expecting that to be removed. I also think we should have one test that is about the fallback to the `img` role

Removed `queryFallback` here: b2399de83b33c306eba4b748c6d24a23aeedb192
Added test for `role="img"` fallback here: 11171e28206bfeda76b5182c66de5ab3e692fa9a